lib/sqlHandler.js

```diff
@@ -15,6 +15,8 @@ var modelGenerators = {
 }
 
 var MIN_SERVER_VERSION = '5.6.5'
+var DEFAULT_RELATION_DELIMITER = '-'
+var UNDERSCORED_DELIMITER = '_'
```

Comment:
Would probably match this as `UNDERSCORED_RELATION_DELIMITER`

Reply:
Okay, renamed.